src/commands/getAddress.js

```diff
@@ -39,6 +36,8 @@ const cmd: Command<Input, Result> = createCommand(
           if (e.statusCode === 0x6985) {
             throw new UserRefusedAddress()
           }
+        } else if (e && e.message && e.message.match(/cannot open device with path/)) {
+          throw new CantOpenDevice()
```

Comment:
this is not the correct place where this error should be thrown

please do it at the `deviceAccess` level so we benefit this from all over the places.

i think it's basically the failure case of `open()`